source/includes/steps-export.yaml

```diff
@@ -0,0 +1,77 @@
+title: Navigate to the :doc:`collection </collections>` you wish to export data into.
+level: 4
+ref: select-collection
+content: |
+  You can either select the collection from the
+  :doc:`Collections </collections>` tab or click the collection in the
+  left-hand pane.
+
+---
+title: Click :guilabel:`Collection` in the top-level menu and select :guilabel:`Export Collection`
+level: 4
+ref: export-collection
+content: |
+
+  .. raw:: html
+
+     <br>
+
+  .. figure:: /images/compass/export-data-option-select.png
+     :alt: Export data select menu
+
+  .. raw:: html
+
+     <br>
+
+  |compass-short| displays the following dialog:
+
+  .. raw:: html
+
+     <br>
+
+  .. figure:: /images/compass/export-data-dialog.png
+     :alt: Export data dialog
+
+  The top section of the export dialog displays the query
+  entered in the query bar prior to export, if applicable. You
+  can use this query to filter which collection documents are
+  exported. If no query was specified, this section displays
+  ``undefined``.
+
+---
+title: Use the :guilabel:`Export Full Collection` toggle to indicate 
+  whether to export only the documents matched by the query or the
+  full collection.
+level: 4
+ref: export-full-collection
+content: |
+    Having an ``undefined`` query filter and toggling the
```

Comment:
Two things here - so I actually think it's easier to read in note form. It looks a little scrunched as is and tough to scan. Sorry for going back and forth on this.

Also, I think we can phrase this to be more active voice. Consider something like:

```
|compass-short| exports the full collection both when the query filter is ``undefined`` and when you enable the
:guilabel:`Export Full Collection` option.
```

Reply:
actually, maybe say 

```
|compass-short| exports the full collection when the query filter is ``undefined``, or when you enable the :guilabel:`Export Full Collection` option.
```

my initial suggestion makes it sound like you need both to be true.

Reply:
Good catch! Definitely easier to read now.